lib/src/pause_event.dart

```diff
@@ -25,6 +25,13 @@ VMPauseEvent newVMPauseEvent(Scope scope, Map json) {
 
 /// An event indicating that an isolate has been paused or resumed.
 abstract class VMPauseEvent {
+  /// Is the isolate paused at an await, yield, or yield* statement?
```

Comment:
Nit: 
Tweak comment to 
/// Whether the isolate is paused at an await, yield, or yield* statement.

side note: a yield statement is an async suspension? I thought that was sync although I haven't used yield in my code.

Reply:
I'll update the text tomorrow. The text was taken from [here](https://github.com/dart-lang/sdk/blob/master/runtime/vm/service/service.md#event) and includes yield. I don't know if the naming is accurate, but I guess the expectation is the same when a user hits "step over" on a yield - it should pause on the next statement in their code, even though though there was other execution in between.